docs/concepts/09_glossary.md

```diff
@@ -0,0 +1,157 @@
+SSI eliminates the need for intermediaries and provides individuals with greater privacy, security, and control over their personal data.
+It is an emerging concept that has the potential to transform how identity is managed and verified across various sectors.
+
+### Verifiable Credentials
+
+Verifiable credentials are digital credentials that can be used to prove claims about a person, organization, or thing, and are designed to be portable, interoperable, and privacy-preserving.
+Verifiable credentials are often associated with DIDs and can be stored and managed using a DID-based identity system.
+
+## KILT Protocol Specific Terms
+
+### Attestation
+
+The act of providing evidence or confirming the validity of a claim, typically performed by a trusted attester.
+
+### Attester
+
+A person, organization or entity that verifies and validates the identity or credentials of an individual in a digital or physical setting.
+Attesters play a critical role in building trust and establishing the authenticity of information and data shared between parties.
+
+### Claimer
+
+An individual or entity that asserts a claim or statement about their identity or qualifications.
+The claimer can use digital credentials, such as badges or certificates, to provide evidence of their claims, which can be verified by third-party entities or systems.
+
+### Claim Type (CType)
+
+A specific type of claim that can be made about an individual, such as their education, work experience, or identity information.
+Each claim type has a defined set of attributes that must be provided to support the claim, and it can be used to generate verifiable credentials that can be shared with others.
+
+### Credential
+
+A verifiable digital representation of a claim made by a claimer, which has been attested to by a trusted entity, such as an attester.
+It consists of a set of attributes that describe the claim and the proof of its validity, and can be shared with third parties to provide verifiable proof of the claim.
+
+### Digital Identity
+
+A self-sovereign identity that is owned and controlled by the individual or entity it represents. It consists of verifiable credentials that are issued by trusted entities, such as attesters, and can be used to prove claims about the individual or entity's identity, qualifications, or other attributes.
+
+### KILT Coin
+
+The native coin of the KILT Protocol network, used for staking, transaction fees and as a means of exchange on the network.
+
+### KILT Protocol
+
+An open-source blockchain protocol for issuing self-sovereign, and verifiable credentials for Web3, the next generation of the Internet.
+KILT’s mission is to return control over personal data to its owner, restoring privacy to the individual.
+Developers and entrepreneurs can use KILT’s SDK to quickly spin up applications for issuing, holding and verifying credentials in order to create businesses around identity and privacy.
```

Comment:
i would remove that sentence. Sounds like we have the businesses side in mind and not the users.